src/elements/element.line.js

```diff
@@ -43,17 +40,17 @@ module.exports = Element.extend({
 		ctx.save();
 
 		// Stroke Line Options
-		ctx.lineCap = vm.borderCapStyle || globalOptionLineElements.borderCapStyle;
+		ctx.lineCap = valueOrDefault(vm.borderCapStyle, globalOptionLineElements.borderCapStyle);
 
 		// IE 9 and 10 do not support line dash
 		if (ctx.setLineDash) {
 			ctx.setLineDash(vm.borderDash || globalOptionLineElements.borderDash);
```

Comment:
shouldn't this line be converted to `valueOrDefault` too?

Reply:
I can make that change